test/e2e/files/nri-resource-policy-balloons-deployment.yaml.in

```diff
@@ -66,6 +66,7 @@ spec:
             - /tmp/nri-resource-policy.pid
             - -metrics-interval
             - 5s
+            - --cache-output-enable
```

Comment:
nitpick: same nitpick, this sort of sticks out as 'I can't guess what this enables'...